sklearn/discriminant_analysis.py

```diff
@@ -556,6 +558,8 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
     .. versionchanged:: 0.17
        Deprecated :class:`qda.QDA` have been moved to *QuadraticDiscriminantAnalysis*.
```

Comment:
No no, I was asking if you could change the `*Quadra....` to `:class:`Quad....`, so that it would be shown as a link.